homeassistant/components/bring/todo.py

```diff
@@ -215,3 +238,33 @@ async def async_delete_todo_items(self, uids: list[str]) -> None:
             raise HomeAssistantError("Unable to delete todo item for bring") from e
 
         await self.coordinator.async_refresh()
+
+    async def async_send_message(
+        self,
+        message: BringNotificationType,
+        item: str | None = None,
+    ) -> None:
+        """Send a push notification to members of a shared bring list."""
+
+        try:
+            await self.coordinator.bring.notify(self._list_uuid, message, item or None)
+        except BringRequestException as e:
+            raise HomeAssistantError(
+                translation_domain=DOMAIN,
+                translation_key="notify_request_failed",
+            ) from e
+        except ValueError as e:
+            raise ServiceValidationError(
+                translation_domain=DOMAIN,
+                translation_key="notify_missing_argument_item",
+            ) from e
+        except TypeError as e:
+            raise ServiceValidationError(
+                translation_domain=DOMAIN,
+                translation_key="notify_invalid_message_type",
+                translation_placeholders={
+                    "notification_types": ", ".join(
+                        x.value for x in BringNotificationType
+                    ),
+                },
+            ) from e
```

Comment:
How is this possible considering the schema requires the user to pick one of the 4 notification types?

Reply:
you are right, the schema validator already catches this so this is superflous, will remove it